plugins/modules/docker_compose2.py

```diff
@@ -0,0 +1,866 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+DOCUMENTATION = '''
+
+module: docker_compose2
+
+short_description: Manage multi-container Docker applications with Docker Compose.
+
+author: Léo El Amri (@lel-amri)
+
+description:
+  - Uses Docker Compose to start and shutdown services.
+  - Swarm mode is not supported (thus secrets and configs aren't supported).
```

Comment:
```suggestion
  - Swarm mode is not supported (thus secrets and configs are not supported).
```
(appears some more times in the return value docs)

Reply:
I think I got them all.